README.rst

```diff
@@ -8,113 +8,115 @@ Masu
 
-  # Run the celery worker locally with access to the OpenShift rabbitmq
-  celery -A masu.celery.worker --broker=amqp://localhost:5672// worker
+Run ``oc cluster up`` once before running the ``make`` commands to generate the referenced config file.
 
-  # To clean up your development enviornment
-  make oc-rm-dev
+Tab Completion
+**************
+The Openshift CLI does offer shell/tab completion. It can be generated for either bash/zsh and is available by running `oc completion bash|zsh` The following example generates a shell script for completion and sources the file.  ::
 
+    oc completion zsh > $HOME/.oc/oc_completion.sh
+    source $HOME/.oc/oc_completion.sh
 
-To gain temporary access to the database within OpenShift, port forwarding is used. ::
+Access to the Koku DB
+*********************
+To gain direct access to the Koku database running on OpenShift, port forwarding must be used. ::
 
-  # Port forward to 15432
+  # Forward port 5432 on the database pod to localhost:15432
   make oc-forward-ports
 
+  # Access the DB
   psql koku -U kokuadmin -p 15432 -h localhost
 
-  # Stop port forwarding
+  # Stop forwarding
   make oc-stop-forwarding-ports
 
 
 Testing and Linting
 -------------------
 
-Masu uses tox to standardize the environment used when running tests. Essentially, tox manages its own virtual environment and a copy of required dependencies to run tests. To ensure a clean tox environment run ::
-
-    tox -r
+Masu uses ``tox`` to run unit tests. The simplest use case is to run ``tox`` from the top-most directory of the git repository with no additional arguments.
 
-This will rebuild the tox virtual env and then run all tests.
-
-To run unit tests specifically::
+To run only the unit tests ::
 
     tox -e py36
 
-To lint the code base ::
+To run only the linters ::
 
     tox -e lint
 
-During development it can sometimes be useful to unittest a specific module or test class. Running tox can become burdensome when rapidly iterating on tests. An `.env.test` file in the base of the masu repository can be used to quickly modify database environment variables for testing. An example .env.test file::
+During development it can sometimes be useful to unittest a specific module or test class. To do this, create an `.env.test` file in the base of the masu repository. This can be used to modify database environment variables for development or testing.
+
+An example .env.test file::
 
+    MASU_SECRET_KEY='t0ta!!yr4nd0m'
     DATABASE_ENGINE=postgresql
     DATABASE_NAME=test
     DATABASE_HOST=localhost
     DATABASE_PORT=15432
     DATABASE_USER=kokuadmin
     DATABASE_PASSWORD=''
 
-An example workflow for isolated testing might look like the following ::
+An example workflow for isolated testing ::
 
     ./tests/create_db.sh
     source .env.test
```

Comment:
Maybe include something like `python -m unittest masu.tests.test_module.TestClass.test`?

Reply:
That statement is 2 lines below this comment.

`python -m unittest tests.module.TestClass`